letsencrypt/plugins/manual_test.py

```diff
@@ -71,25 +71,29 @@ def test_perform_test_command_oserror(self, mock_popen):
         mock_popen.side_effect = OSError
         self.assertEqual([False], self.auth_test_mode.perform(self.achalls))
 
+    @mock.patch("letsencrypt.plugins.manual.socket.socket", autospec=True)
```

Comment:
I hadn't known about the ability to autospec the mock in the decorator before.  Seems like a good thing to add to just about all of our tests. :+1: